src/dataflow/components/dataflow-program.tsx

```diff
@@ -169,23 +167,23 @@ export class DataflowProgram extends BaseComponent<IProps, IState> {
               ref={(elt) => this.editorDomElement = elt}
             >
               <div className="flow-tool" ref={elt => this.toolDiv = elt}/>
+              {showZoomControl &&
```

Comment:
Similarly, we show the zoom controls on a complete relay program (not editable).